wiki/Tournaments/Zotcade/2019/en.md

```diff
@@ -0,0 +1,294 @@
+- Tiebreaker
+  - **[DJ Sharpnel - Blue Army (Night Bunny 7) [IcyWorld's Lv.10]](https://osu.ppy.sh/beatmapsets/657152#mania/1392059)**
+
+#### Round 2
+
+- NoMod
+  - [Rhodanthe\* - Happy\*Pretty\*Clover (Rivals_7) [4K Insane Lv. 20]](https://osu.ppy.sh/beatmapsets/583943#mania/1235505)
+  - [TrySail - TrySail Selection 2 (ALEFY) [BraveSail (short Ver.)]](https://osu.ppy.sh/beatmapsets/661108#mania/1399712)
+  - [Kyary Pamyu Pamyu - Ninja Re Bang Bang (ecafree2) [4K HD]](https://osu.ppy.sh/beatmapsets/372942#mania/1728348)
+  - [TAG underground - POSSESSION (maskeara) [CHALLENGE]](https://osu.ppy.sh/beatmapsets/551469#mania/1168087)
+  - [Halozy - GO 4 IT (Virtue-) [Believe Yourself]](https://osu.ppy.sh/beatmapsets/638795#mania/1354962)
+  - [Nanahira - Nanahira Pack Vol.2 (PianoLuigi) [You make my life 1UP!]](https://osu.ppy.sh/beatmapsets/793828#mania/1667317)
+  - [Yooh - FIRE FIRE -DARK BLAZE REMIX- (PiraTom) [EXHAUST]](https://osu.ppy.sh/beatmapsets/227106#mania/530196)
+  - [Qrispy Joybox feat.Sana - Twinkle Wonderland (AchsanLovers) [Wonderful]](https://osu.ppy.sh/beatmapsets/587904#mania/1244809)
+  - [RoughSketch feat.Aikapin - Grimm (Kobaryo's FTN-Remix) (CHARGE) [Comet's Hard]](https://osu.ppy.sh/beatmapsets/597690#mania/1271990)
+- Tiebreaker
+  - **[Minase Inori - MELODY FLAG (Amiichii) [4K Melody]](https://osu.ppy.sh/beatmapsets/629560#mania/1399580)**
+
+#### Round 1
+
+- NoMod
+  - [jioyi attacked by mirror - I lost my flp (Madoka2574) [Normal]](https://osu.ppy.sh/beatmapsets/818926#mania/1720628)
+  - [ginkiha - Starlight Express (bakabakemono) [ADVANCED Lv.12]](https://osu.ppy.sh/beatmapsets/216238#mania/507625)
+  - [Camellia - Routing ([ A v a l o n ]) [Chickening's Hard]](https://osu.ppy.sh/beatmapsets/414833#mania/983252)
+  - [Triple Booking - Yamato Nadeshiko Education (- Bad Apple -) [MX 'Shino']](https://osu.ppy.sh/beatmapsets/283727#mania/641427)
+  - [Satoshi with Pikachu (CV:Rica Matsumoto/Otani Ikue) - Alola!! (Kuo Kyoka) [Solgaleo with Lunala / 4K Hard]](https://osu.ppy.sh/beatmapsets/538085#mania/1139998)
+  - [lapix - Debug Dance (Syadow-) [Amii's ORANGE]](https://osu.ppy.sh/beatmapsets/688459#mania/1689096)
+  - [Ni-Sokkususu - Shukusai no Elementalia ([Emiria]) [Avalon's Hard]](https://osu.ppy.sh/beatmapsets/440089#mania/958619)
+- Tiebreaker
+  - **[MC Mental - MC MENTAL @ HIS BEST (hi19hi19) [Hard]](https://osu.ppy.sh/beatmapsets/503710#mania/1071695)**
+
+## Ruleset
+
+![iDOL@NSTER Bracket Format](img/format.png)
+
+Any disregard for these rules will be met with punishment, ranging from a warning to immediate disqualification. It will be up to the discretion of the staff to punish based on the severity of the offence. **These rules and regulations are subject to change during the event.**
+
+### Lobby and Matches
+
+1. Missing your match counts as a loss; no rematches will be made.
+1. Score V2 will be used as the win condition.
```

Comment:
We increment according to ASC. Section was deleted on recent ASC PR for some reason :(.
![](https://puu.sh/Dtmue/8f39e6f629.png)

Reply:
is that just an example or a rule (also why, doesn't that make it harder to update?)

Reply:
> The Article Styling Criteria (ASC) serves as the osu! wiki's enforced styling standards to keep **consistency** in **clarity**, **formatting**, and **layout** in all articles.

It should be considered a rule or preferred to fulfill ASC's goal. Anyway, it has been used in most articles and doesn't confuse new people about this behavior. It also keeps track of numbers when it really matters (i.e. referencing a step, but having to go to parser instead to get said number).